HermaFx.DataAnnotations/MinAttribute.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Globalization;
+
+namespace HermaFx.DataAnnotations
+{
+	/// <summary>
+	/// Can be used with parameter target..
+	/// </summary>
+	[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]
```

Comment:
Should we add Parameter too?

Reply:
Parameter is already added, no? (Property, Field, Parameter)